printer_mods/hartk1213/Voron2.4_Pins_Mod/README.md

```diff
@@ -36,6 +44,11 @@ here is what the XY Joint will look like
 ![XY Joint](Images/1.png) 
 
 
+There is a new file for the A Drive to go along with [Arkeets MGN12](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/arkeet/mgn12) mod that has the X endstop on the toolhead 
+
+the files for the endstop relocation mod are here [Voron2.4_Y_Endstop_Relocation](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/hartk1213/Voron2.4_Y_Endstop_Relocation)
```

Comment:
Change to relative link

    [Voron2.4_Y_Endstop_Relocation](../Voron2.4_Y_Endstop_Relocation)